src/main/java/com/spotify/reaper/storage/CassandraStorage.java

```diff
@@ -739,4 +779,78 @@ private RepairRun buildRepairRunFromRow(Row repairRunResult, long id){
         .build(id);
   }
 
+  @Override
+  public boolean takeLeadOnSegment(long segmentId) {
+    Row lwtResult = session.execute(getLeadOnSegmentPrepStmt.bind(segmentId, ReaperApplication.REAPER_INSTANCE_ID, ReaperApplication.getInstanceAddress())).one();
+    if (lwtResult.getBool("[applied]")) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "[applied]" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)